README.md

```diff
@@ -0,0 +1,219 @@
+
+### Stubborn
+
+My least favorite bug. When I've got a strong opinion, my tendency is to dig in and defend it. This is most problematic if I transition away from being open-minded in a discussion. However, I love a good argument and am always pleased to have my mind changed as it usually means I'm seeing the world with more truth. If you find me pushing back against you, lean into it and help me to hear you. If you ever find me being overly stubborn, point it out; it won't be a surprise although you may catch me getting red from embarrassment.
+
+### Gets bored easily
+
+I prefer to move fast and I feed off progress. When this isn't happening, my brain will push me towards something new. This can occur on a variety of fronts: projects, books, and even conversations. A consequence of this tendency is that I am generally better at starting than finishing. When projects drag and lose momentum my interest tends to go with it. If you manage me, helping to keep new challenges in front of me is the best way to keep me happy. If you report to me or are a peer, you'll find I'm at my best when I can step back and let you do your thing after I've put in enough energy to foster escape velocity.
+
+## My Story
+
+I've traversed an atypical path into Engineering Management. Here's my lifestory in a nutshell to provide you some insight into my road from then to now.
+
+### Chapter 1: NOVA
+
+- I grew up outside of Washington, DC in a nice corner of Northern Virginia.
+- In my youth, I spent a lot of my time playing soccer and videogames and reading. My Dad worked for DoD so we had a computer in my house about as early as I can remember.
+- I first began writing code at the age of 12 because I wanted to build a website for my Diablo guild. It had a lot of animated .gifs and I thought it was the coolest.
+- In high school, I was fortunate that my school had computer science courses and I began programming in C++ in 10th grade. I talked my mom into getting two whiteboards for my bedroom so I could plan out programs at home.
+- I played lacrosse for a state championship-winning team and made All-State playing bass clarinet.
+
+### Chapter 2: State College
+
+- I enrolled at Penn State University to study Computer Science. It was my dream to open my own videogame studio as a technical founder.
+- About two years in, I got freaked out about being pigeon-holed as a programmer in a cubicle for the rest of my life. In a moment of panic, I switched my major to Electrical Engineering (EE).
+- Focused my EE degree on antenna design, lasers, and optics.
+- Loved being in college so much that I stuck around for half of a super-duper senior year. I think anyone who only goes for a fifth year of undergrad isn't truly dedicated to the cause.
```

Comment:
LOL

Reply:
I got confused here too, I think(?) you meant to say: `I think anyone who only goes for a **fourth** year of undergrad isn't truly dedicated to the cause.`

Reply:
I went to school for 5.5 years and intended to joke that anyone who only did 5.0 years was a try-hard.

4th year = Senior
5h year = Super Senior
6th year = Super Duper Senior

This is the most important thing I've done today. 😆